var/spack/repos/builtin/packages/ravel/package.py

```diff
@@ -14,13 +14,14 @@ class Ravel(CMakePackage):
     url = 'https://github.com/llnl/ravel/archive/v1.0.0.tar.gz'
 
     version('1.0.0', sha256='e1e1ac6d70c9aae915623d81a8f1258488fd26f880612fe21f2e032827aa93eb')
+    patch('qpainterpath.patch')
```

Comment:
Can you add a comment:
```suggestion
    # See https://github.com/LLNL/ravel/pull/18
    patch('qpainterpath.patch')
```
pointing at your PR upstream?